pkg/skaffold/util/util.go

```diff
@@ -227,3 +229,15 @@ func AbsFile(workspace string, filename string) (string, error) {
 	}
 	return filepath.Abs(file)
 }
+
+// NonEmptyLines scans the provided input and returns the non-empty strings found as an array
+func NonEmptyLines(input []byte) []string {
+	var result = []string{}
+	scanner := bufio.NewScanner(bytes.NewReader(input))
+	for scanner.Scan() {
+		if line := scanner.Text(); len(line) > 0 {
+			result = append(result, line)
+		}
+	}
+	return result
```

Comment:
You should handle `scanner.Err()` here

Reply:
Suggestions on what should happen?  The input is a bag o' bytes rather than real I/O.

Reply:
Maybe just ignore my comment then.